src/main/scala-sbt-1.0/fm/sbt/CrossSbtUtils.scala

```diff
@@ -0,0 +1,7 @@
+package fm.sbt
+
+import sbt.librarymanagement.RawRepository
+
+object CrossSbtUtils {
+  def rawRepository(resolver: S3URLResolver, name: String) = new RawRepository(resolver, name)
```

Comment:
I'd be tempted to just make the signature:

`def makeRawRepository(resolver: S3URLResolver): RawRepository = new RawRepository(resolver, resolver.name)`

Is the `CrossSbtUtils` convention something you're seeing in other plugin projects for helper methods?

Reply:
See:
https://github.com/vmunier/sbt-web-scalajs/pull/38
http://www.scala-sbt.org/1.0/docs/Migrating-from-sbt-013x.html#sbt+version+specific+source+directory